lib/collector/serverless.js

```diff
@@ -35,10 +38,8 @@ class ServerlessCollector {
       agent_version: agent.version
     }
     this.payload = {}
-    this.pipePath = (agent.pipePath || process.env.pipePath) ?
-      (agent.pipePath || process.env.pipePath) :
-      '/tmp/newrelic-telemetry'
-    this.pipeError = false
+    this.NEWRELIC_PIPE_PATH = nrPipePath || process.env.NEWRELIC_PIPE_PATH || defaultPipePath
```

Comment:
nit: stylistically, we typically only use SNAKE_CASE for constants / env vars.

but yeah, that's something better handled via automated tools and not PR nit picks.

similarly to prior, the "new relic" part on the member variable is somewhat redundant. (unlike the ENV var which gets set outside of the agent).